src/convert.ts

```diff
@@ -73,43 +147,88 @@ export function toTs(node: Flow | TSType): TSType {
     case 'TSUndefinedKeyword':
     case 'TSUnionType':
     case 'TSVoidKeyword':
+    // @ts-ignore
+    case 'TSTypeAnnotation':
+    // @ts-ignore
     case 'TSTypeParameterDeclaration':
+    // @ts-ignore
     case 'TSAsExpression':
+    // @ts-ignore
     case 'TSPropertySignature':
       return node
 
+    case 'TypeAnnotation':
+      return tsTypeAnnotation(toTsType(node))
+
+    case 'InterfaceDeclaration':
+      const { properties, spreads } = objectTypeAnnotationPropertiesAndSpreads(
+        node.body
+      )
+      if (spreads.length) {
+        throw new Error('Spreads in interfaces unsupported')
+      }
+      return tsInterfaceDeclaration(
+        node.id,
+        node.typeParameters
+          ? tsTypeParameterDeclaration(
+              node.typeParameters.params.map(toTsTypeParameter)
+            )
+          : null,
+        node.extends && node.extends.length
+          ? node.extends.map(
+              (_: InterfaceExtends): TSExpressionWithTypeArguments => toTs(_)
```

Comment:
Why do you need the explicit annotations here?
```suggestion
              toTs
```